.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -96,6 +96,21 @@ jobs:
           command: test
           args: --no-default-features
 
+  kani:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Verify with Kani
+        uses: model-checking/kani-github-action@02764e122079011adaf31e7d0630a8841d277507
+        with:
+          args: |
+            --tests -p prost-types --default-unwind 3 \
```

Comment:
Where does the value 3 come from?

Reply:
This was the minimum unwind number that allowed this harness to work. 

The code that is triggering unrolling seems to be from the standard library, not customer code. CBMC is fairly loose on what it chooses to unroll and it sometimes unrolls code in unreachable functions. This was less of an issue in summer 2022 because std support was not yet implemented.

Reply:
127d7d1d9b3120aef0c8818ac16f105464ab298c: I added comment clarifying this unwind value's meaning. This should help with customer-side adjustments if this harness stops working because the minimum required unwind number went up by some reasonable number.